### PR TITLE
Build against 23.08 runtimes

### PR DESCRIPTION
**Pull request by akurtakov**

### CODE REVIEW

**Comment by akurtakov** [comment]
@bbhtt Is there a chance you can check what's wrong here? It fails for me with:
```
bsdtar: viber.deb: Not found in archive
bsdtar: Error exit delayed from previous errors.
tar: data.tar.xz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
mv: cannot stat 'opt/viber': No such file or directory
sed: can't read usr/share/applications/viber.desktop: No such file or directory
install: cannot stat 'usr/share/viber/16x16.png': No such file or directory
install: cannot stat 'usr/share/viber/24x24.png': No such file or directory
install: cannot stat 'usr/share/viber/32x32.png': No such file or directory
install: cannot stat 'usr/share/viber/48x48.png': No such file or directory
install: cannot stat 'usr/share/viber/64x64.png': No such file or directory
install: cannot stat 'usr/share/viber/96x96.png': No such file or directory
install: cannot stat 'usr/share/viber/128x128.png': No such file or directory
install: cannot stat 'usr/share/viber/256x256.png': No such file or directory
install: cannot stat 'usr/share/icons/hicolor/scalable/apps/Viber.svg': No such file or directory
/app/bin/apply_extra: line 13: /app/extra/viber/qt.conf: No such file or directory
rm: cannot remove 'control.tar.xz': No such file or directory
rm: cannot remove 'data.tar.xz': No such file or directory
        ID                       Branch          Op          Remote                Download
 1. [✗] com.viber.Viber          test            i           viber1-origin         154.2 MB / 155.6 MB

Error: While trying to apply extra data: apply_extra script failed, exit status 256
error: Failed to install com.viber.Viber: While trying to apply extra data: apply_extra script failed, exit status 256
```
but it somehow doesn't make sense to me that the deb file is not in the right place if I change to 23.08 and be there with 22.08

**Comment by bbhtt** [comment]
The syntax for `bsdtar` is different from `ar`. 

Try

```
bsdtar -Oxf viber.deb data.tar.xz | bsdtar --exclude 'usr' -xf -
```

**Comment by bbhtt** [comment]
Actually I saw you're copying the desktop file from the deb too. In that case

```
bsdtar -Oxf viber.deb data.tar.xz | bsdtar -xf -
```

**Comment by akurtakov** [comment]
Are you interested in becoming committer here? I took it after the previous one abandon it and would much appreciate getting help.

**Comment by bbhtt** [comment]
I already have commit access to all repos. I don't use this, but I can help on this one if you want.

File: com.viber.Viber.json [hunk 1]
```diff
@@ -32,15 +32,15 @@
                 "install -Dm755 apply_extra /app/bin/apply_extra",
                 "install -Dm755 viber /app/bin/viber",
                 "install -Dm644 com.viber.Viber.appdata.xml /app/share/appdata/com.viber.Viber.appdata.xml",
-                "cp /usr/bin/ar /app/bin",
+                "cp /usr/bin/bsdtar /app/bin",
                 "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib"
```
Comment (bbhtt) [inline com.viber.Viber.json:36]: Drop this. This was needed for `ar`

File: com.viber.Viber.json [hunk 2]
```diff
@@ -32,15 +32,15 @@
...
             ],
             "sources": [
                 {
                     "type": "script",
                     "dest-filename": "apply_extra",
                     "commands": [
-                        "ar x viber.deb",
+                        "bsdtar x viber.deb",
```
Comment (bbhtt) [inline com.viber.Viber.json:43]: Change to `bsdtar -Oxf viber.deb data.tar.xz | bsdtar -xf -`

Comment (bbhtt) [reply com.viber.Viber.json:43]: Change `rm -r` to `rm -r viber.deb opt usr`

File: com.viber.Viber.json (hunk 1 above)
Comment (akurtakov) [reply com.viber.Viber.json]: Done

**Comment by bbhtt** [comment]
Hey @akurtakov  we fixed a bug in libnotify that was causing applications on Electron baseapp 23.08 to crash https://github.com/flathub/org.electronjs.Electron2.BaseApp/pull/48

If you can trigger a rebuild via https://buildbot.flathub.org/#/ it'd be great.

**Pull request merged**